python/pyxet/src/lib.rs

```diff
@@ -679,7 +945,16 @@ impl PyWFile {
     pub fn writable(&self) -> PyResult<bool> {
         Ok(true)
     }
+
```

Comment:
nit: a comment about why this is needed (pickling?) can be added later

Reply:
@hoytak should have a better answer to question